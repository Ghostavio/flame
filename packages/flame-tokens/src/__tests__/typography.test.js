const { fontSizes, fontSizeAliases } = require('../typography');

describe('fontSizes', () => {
  it('returns list of rem value by size', () => {
    expect(fontSizes).toMatchSnapshot();
  });
});

describe('fontSizeAliases', () => {
  it('returns list of rem value by alias size', () => {
    expect(fontSizeAliases).toMatchSnapshot();
  });
});
