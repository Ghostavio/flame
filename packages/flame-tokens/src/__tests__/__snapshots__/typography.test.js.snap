// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`fontSizeAliases returns list of rem value by alias size 1`] = `
Object {
  "": "1rem",
  "large": "1.125rem",
  "small": "0.875rem",
  "xlarge": "1.5rem",
  "xsmall": "0.75rem",
  "xxlarge": "2.25rem",
  "xxsmall": "0.5rem",
}
`;

exports[`fontSizes returns list of rem value by size 1`] = `
Object {
  "text": "1rem",
  "text-l": "1.125rem",
  "text-s": "0.875rem",
  "text-xl": "1.5rem",
  "text-xs": "0.75rem",
  "text-xxl": "2.25rem",
  "text-xxs": "0.5rem",
}
`;
